oml/functional/metrics.py

```diff
@@ -16,7 +16,6 @@
 def calc_retrieval_metrics(
```

Comment:
There is no info about the structure of distances and mask_gt

Reply:
Also now we need to write 1 extra line with applying everywhere

Reply:
what do you mean? we have a doc string

Reply:
> Also now we need to write 1 extra line with applying everywhere

it's okay because everywhere == 1 place in EmbeddingsMetrics, we don't have other entry points to the metrics in the library

Reply:
We have no info, that `apply_mask_to_ignore` have to be applied to `mask_gt` and `distances` before calling `calc_retrieval_metrics`.

Suppose, our potential users don't want to create `EmbeddingMetrics` with inner accumulator, keys, etc, if they have the necessary tensors and we offer a function with such good naming. 

We can also forget in several months, that `apply_mask_to_ignore` has to be called outside.

Reply:
![Screenshot from 2023-03-05 11-06-59](https://user-images.githubusercontent.com/43877180/222954075-6de3a70d-36c0-4bf9-8819-9a323fcb3a9e.png)
We have ~10 places, where we call `calc_retrieval_metrics` and `apply_mask_to_ignore` should be added everywhere and we need don't forget to add it in the future.

Reply:
To be honest I don't really see why you decided to remove `mask_to_ignore` introducing potential bugs. You can add `apply_mask_to_ignore` before the postprocessor and don't change other lines. Calling `apply_mask_to_ignore` two times is extremely fast and doesn't produce errors.

Reply:
discussed offline